ehr_billing/api-src/org/labkey/api/ehr_billing/pipeline/InvoicedItemsProcessingService.java

```diff
@@ -60,9 +60,11 @@ static InvoicedItemsProcessingService get()
      * Generate invoice number for a billing task processed row and billing period date.
      * @param row the billing task processed row to get values from
      * @param billingPeriodDate a date within a billing period
+     * @param logger log4j logger for errors and warnings
      * @return generated invoice num
      */
-    String getInvoiceNum(Map<String, Object> row, Date billingPeriodDate);
+    @Nullable
+    String getInvoiceNum(Map<String, Object> row, Date billingPeriodDate, Logger logger);
```

Comment:
@labkey-jeckels if this looks ok then I can work with Binal to update wnprc's implementation.

Reply:
Yes, looks OK to me